elasticsearch-vmss/azuredeploy.json

```diff
@@ -0,0 +1,504 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "type": "string",
+      "defaultValue": "es-azure",
+      "metadata": {
+        "description": "The name of the cluster."
+      }
+    },
+    "vmSku": {
```

Comment:
Yes - D1_v2 is fine (e.g. Dv2)...  The param isn't really needed for the scenario and will simplify the example (i.e. less chance of failure).